ScoutSuite/providers/azure/facade/securitycenter.py

```diff
@@ -33,3 +33,21 @@ def __init__(self, credentials, subscription_id):
         except Exception as e:
             print_exception('Failed to retrieve auto provisioning settings: {}'.format(e))
             return []
+
+    async def get_information_protection_policies(self):
+        try:
+            return await run_concurrently(
+                lambda: list(self._client.information_protection_policies.list())
```

Comment:
I'm getting a `securitycenter.py L40: Failed to retrieve information protection policies: list() missing 1 required positional argument: 'scope'` error.

Reply:
I don't have that error on my code. Maybe is fixed now, can you check please?

Reply:
Nope, and it's in the code https://github.com/Azure/azure-sdk-for-python/blob/ccd73c15088f969b8adaaf10f26faf00387405d8/sdk/security/azure-mgmt-security/azure/mgmt/security/operations/_information_protection_policies_operations.py#L170 so maybe you're not running latest version of the sdk?

Reply:
opened an issue - https://github.com/Azure/azure-sdk-for-python/issues/7687